lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -752,9 +784,14 @@ def _rows_for_users(self, context, users, header_row, graded_scorable_blocks):
                     else:
                         earned_possible_values.append(['Not Attempted', problem_score.possible])
 
+            context.task_progress.succeeded += 1
             success_rows.append(
                 student_fields + [enrollment_status, course_grade.percent] + _flatten(earned_possible_values))
 
+        context.task_progress.update_task_state(extra_meta={'step': 'ProblemGradeReport Step 5: Calculating Grades'})
+        log_message = u'{0} {1}/{2}'.format('ProblemGradeReport Step 5: Calculating Grades',
```

Comment:
can we log it after processing 100/150/200 records rather than after every record?

Reply:
Thanks to the batching logic, we are now logging after every batch is processed (log is outside _for-loop_). 
This means that the log will be made after a batch of **100 users** (batch size set) has been processed.

Reply:
Got it.